requirements-olmo.txt

```diff
@@ -36,4 +36,6 @@ immutabledict
 black
 flake8
 isort
-autoflake
\ No newline at end of file
+autoflake
+tenacity==8.4.1
```

Comment:
could we also add to `requirements.txt` if possible?

Reply:
Done!